src/components/BlogDeleteModal.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import { Modal, ModalBody, ModalFooter, ModalHeader, Button } from 'reactstrap';
+
+import { csrfFetch } from '../util/CSRF';
+
+class BlogDeleteModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOpen: Boolean(props.isOpen),
+    };
+
+    this.close = this.close.bind(this);
+    this.confirm = this.confirm.bind(this);
+    this.keyPress = this.keyPress.bind(this);
+  }
+
+  close() {
+    document.removeEventListener('keyup', this.keyPress);
```

Comment:
No need for an event listener on the whole document. You can add a `componentDidUpdate` method that uses a ref to put focus on the delete button when the modal opens. That avoids the cleanup problems.